library/src/main/java/com/getstream/sdk/chat/rest/request/BanUserRequest.java

```diff
@@ -41,4 +41,29 @@ public BanUserRequest(@NotNull String targetUserId, @Nullable Integer timeout,
         this.channelType = channelType;
         this.channelId = channelId;
     }
+
+    @NotNull
+    public String getTargetUserId() {
+        return targetUserId;
+    }
+
+    @Nullable
+    public Integer getTimeout() {
+        return timeout;
```

Comment:
Should return `int` instead of `Integer`. The code is spoiled with boxed types. Feel free to fix it also if you touch such code.

Reply:
We use Integer there because we don't need to send `timeout` if it's not set. If change it to `int` timeout will be sent as 0.

Reply:
So `null` is valid value in this case? :\

Reply:
yes

Reply:
So look ugly. But ok then.